tests/lora/test_lora_layers_cogvideox.py

```diff
@@ -126,6 +127,10 @@ def get_dummy_inputs(self, with_generator=True):
         return noise, input_ids, pipeline_inputs
 
     @skip_mps
+    @unittest.skipIf(
+        torch.device(torch_device).type == "cpu" and is_torch_version(">=", "2.5"),
+        "Test not supported on PyTorch 2.5 and CPU.",
+    )
```

Comment:
@a-r-r-o-w not important for this PR but any reason this had to be rewritten for Cog?